### DISCUSSION TITLE
Should we directly zip the output files from simona

### DISCUSSION THREAD

**Discussion by danielfeismann** [ie3-institute/simona#781] [Ideas]
Since I just realised that pandas 'read_csv' also supports zip compression it's quite easy and convinient to implement zipped csv data in analysis of simulations. I guess this would be as well easy to implement into pypsdm though. Thus IMHO it would be a good idea to add compression to the simona outputs. 

But there might be issues I'm not aware of. So lets discuss, what do you think @sebastian-peter @jo-bao @t-ober?

**Comment by sebastian-peter** [comment]
Sounds good. Another way to save space would be to use databases, because they use efficient data types besides just text.

**Reply by danielfeismann** [reply]
True. Doing both or offering both options is also fine for me :) .

**Comment by t-ober** [comment]
Might be worth a try. I don't see any obvious downsides.

**Comment by danielfeismann** [comment]
closed by #965
